### DISCUSSION TITLE
show realistic password strength

### DISCUSSION THREAD

**Discussion by davidak** [tosdr/edit.tosdr.org#1034] [Ideas]
for example with [zxcvbn](https://blogs.dropbox.com/tech/2012/04/zxcvbn-realistic-password-strength-estimation/)

instead of password policy other than length.

**Comment by michielbdejong** [comment]
Hi @davidak, welcome to our little community! :)
Your plan sounds good, maybe you could use https://github.com/bitzesty/devise_zxcvbn for this? Do you want to work on this yourself, or are you suggesting it for other to work on?

**Comment by davidak** [comment]
@michielbdejong Thanks! (I know and like the TOS:DR website for many years)
Looks like a good beginner task. Unfortunately i already work on too many projects. So this is only a suggestions.
